python/lsst/obs/base/ingest_tests.py

```diff
@@ -246,10 +246,14 @@ def testCopy(self):
     def testHardLink(self):
         try:
             self._ingestRaws(transfer="hardlink")
-            self.verifyIngest()
-        except PermissionError as err:
+            # Running ingest through the Click testing infrastructure causes
+            # the original exception indicating that we can't hard-link
+            # on this filesystem to be turned into a nonzero exit code, which
+            # then trips the test assertion.
+        except AssertionError as err:
```

Comment:
Maybe we could list both errors here then so we don't need to worry about it again?